pkg/services/files/uploader.go

```diff
@@ -89,6 +90,24 @@ func newS3Uploader() *S3Uploader {
 	}
 }
 
+type filePathVar struct {
+	fileName   string
+	uploadPath string
+}
+
+func worker(wg *sync.WaitGroup, queue <-chan filePathVar, u *S3Uploader) {
+	defer wg.Done()
+
+	for p := range queue {
+		fname := p.fileName
+		uploadDest := p.uploadPath
+		_, err := u.UploadFile(fname, uploadDest)
```

Comment:
```suggestion
		_, err := u.UploadFile(p.filename, p.uploadDest)
```

Or just pass p right away, but that's a major refactor

Reply:
I agree that P is a major refactor, however I changed this fucntion call to use the "p." notation you suggested